tests/integration/gcp.py

```diff
@@ -440,27 +438,25 @@ def _get_image(self, project, image_name):
         self.logger.debug(f"Looking for {image_name=} in {project=}")
         try:
             response = (
-                self._compute.images().get(project=project, image=image_name,).execute()
+                self._compute_images.get(project=project, image=image_name,)
             )
-            image = response.get("selfLink")
+            image = response.self_link
             return image
         except:
             self.logger.debug(f"Failed to find image in {project=}")
             return None
 
     def _wait_until_reachable(self, hostname):
-        self.logger.info(f"Waiting for {hostname} to respond to ping ...")
-        while True:
-            response = os.system("ping -c 1 " + hostname)
+        self.logger.info(f"Waiting for {hostname} to respond...")
+        i=0
+        while i<20:
```

Comment:
A python linter would complain about it. There should be spaces between the operands:
```
i = 0
while i < 20:
```